WireDomain/Sources/WireDomainSupport/Sourcery/generated/AutoMockable.generated.swift

```diff
@@ -24,15 +24,9 @@
 // swiftlint:disable line_length
 // swiftlint:disable variable_name
 
-public import Foundation
-#if os(iOS) || os(tvOS) || os(watchOS)
-public import UIKit
-#elseif os(OSX)
-public import AppKit
-#endif
-
-public import WireAPI
-public import WireDataModel
+
```

Comment:
Sorry I'm a bit new to sorcery configuration! 

So here we see that we no longer import UIKit. To have this imported we would have to include it in the `autoMockablePublicImports`. Is that correct?

Reply:
Yes, it's correct. 
I thought it doesn't make sense to import `UIKit` everywhere, so from now on we have to explicitly specify it.
Or do you have concerns about this?

Reply:
`UIKit` seems to be needed only in WireFoundationSupport currently.